src/main/java/rx/schedulers/NewThreadScheduler.java

```diff
@@ -16,28 +16,18 @@
 package rx.schedulers;
 
 import rx.Scheduler;
-import rx.internal.schedulers.NewThreadWorker;
-import rx.internal.util.RxThreadFactory;
 
 /**
- * Schedules work on a new thread.
+ * @deprecated This type was never publicly instantiable. Use {@link Schedulers#newThread()}.
  */
+@Deprecated
 public final class NewThreadScheduler extends Scheduler {
```

Comment:
Should be fine for even patch release, `NewThreadScheduler` has no public fields nor methods + private constructor. Without reflection or unchecked casting it's unusable for users.

I vote for removing it right in this PR.